build.sc

```diff
@@ -114,11 +115,13 @@ class JsonModule(val crossScalaVersion: String) extends Fs2DataModule with Cross
       developers = Seq(fs2DataDeveloper)
     )
 
-    object test extends Fs2DataTests {
-      def moduleDeps = Seq(circe)
-    }
+  object test extends Fs2DataTests {
+    def moduleDeps = Seq(circe(outer.crossScalaVersion))
+  }
+
+  object circe extends Cross[CirceModule](scala212, scala213)
 
-  object circe extends Fs2DataModule with PublishModule {
+  class CirceModule(val crossScalaVersion: String) extends Fs2DataModule with CrossScalaModule with PublishModule {
```

Comment:
It should not be necessary, as this module is in a cross module, it will be published for all cross versions.

Reply:
True, reverted. Had some problems when setting up the cross sources for the literals and that change seemed to fix them, not sure what was going on.